Frameworks/CoreText/DWriteWrapper.mm

```diff
@@ -36,14 +41,16 @@
 static IWLazyClassLookup s_lazyUIFont("UIFont");
 static const float c_defaultSystemFontSize = 15.0f;
 
-static ComPtr<IDWriteFactory> CreateDWriteFactoryInstance() {
+// Private helper for creating a DWriteFactory
+static ComPtr<IDWriteFactory> __CreateDWriteFactoryInstance() {
```

Comment:
>__ [](start = 30, length = 2)

Note: Proposing a naming convention here for CT/DW stuff:

two underscores for file-private
one underscore for framework-private

CF I think used a similar pattern, and I think it makes things a little easier to read.